university-service/src/main/kotlin/protobuf/GRPCService.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package protobuf;
+
+import "ProtoMessage.proto";
+
+service TransactionsService{
```

Comment:
you shouldn't copy the same proto in each module. proto files should be placed in separate module and other module can add it as dependency